net/instaweb/rewriter/rewrite_options.cc

```diff
@@ -1578,6 +1579,10 @@ void RewriteOptions::AddProperties() {
       kDirectoryScope,
       NULL, true);  // TODO(jmarantz): implement for mod_pagespeed.
   AddBaseProperty(
+      false, &RewriteOptions::follow_flushes_, "ff", kFollowFlushes,
```

Comment:
The reasons for defaulting this flag to false:
- For very chunky html responses a default of true could hurt
- I understood from @jmarantz that the code that hides under FlushHtml is pretty sensitive. 
What do  you think?